homu/server.py

```diff
@@ -29,8 +29,16 @@ def index():
 
 @get('/queue/<repo_label>')
 def queue(repo_label):
-    repo = g.repos[repo_label]
-    pull_states = sorted(g.states[repo_label].values())
+    if repo_label == 'all':
+        labels = list(g.repos.keys())
```

Comment:
Ah, interesting. 

I'd agree that having no `list` is the right solution, if only because I like to pretend that it will be faster.